src/lib/controllers/tokenize-controller.js

```diff
@@ -12,8 +13,15 @@ export default class TokenizeController {
    */
   static get tokenizeMethods () {
     return {
-      simpleLocalTokenizer: SimpleLocalTokenizer.tokenize.bind(SimpleLocalTokenizer),
-      alpheiosRemoteTokenizer: AlpheiosRemoteTokenizer.tokenize.bind(AlpheiosRemoteTokenizer)
+      simpleLocalTokenizer: {
+        method: SimpleLocalTokenizer.tokenize.bind(SimpleLocalTokenizer),
+        hasOptions: false
+      },
+      alpheiosRemoteTokenizer: {
+        method: AlpheiosRemoteTokenizer.tokenize.bind(AlpheiosRemoteTokenizer),
+        hasOptions: true,
+        uploadOptionsMethod: this.uploadDefaultRemoteTokenizeOptions.bind(this)
+      }

```

Comment:
added check and uploadOptins method to define if we need to upload tokenize options from the remote source